Config/DefaultInput.ini

```diff
@@ -0,0 +1,101 @@
+[/Script/Engine.InputSettings]
```

Comment:
this is marked as a conflict but it not conflicting with anything. don't know what to do regarding this